invenio_rdm_records/services/pids/providers/datacite.py

```diff
@@ -205,26 +176,21 @@ def delete(self, pid, record, **kwargs):
         Otherwise, also it's deleted also remotely.
         :returns: `True` if is deleted successfully.
         """
-        # PIDS-FIXME: move to async task, exception handling included
-        try:
-            if pid.is_reserved():  # Delete only works for draft DOIs
-                if self.is_api_client_setup:
-                    self.api_client.delete_doi(pid.pid_value)
-                else:
-                    logging.warning("DataCite client not configured. " +
-                                    f"Cannot delete DOI for {pid.pid_value}")
-            elif pid.is_registered():
-                if self.is_api_client_setup:
-                    self.api_client.hide_doi(pid.pid_value)
-                else:
-                    logging.warning("DataCite client not configured. " +
-                                    f"Cannot delete DOI for {pid.pid_value}")
-        except DataCiteError as e:
-            logging.warning("DataCite provider errored when deleting " +
-                            f"DOI for {pid.pid_value}")
-            self._log_errors(e)
-
-            return False
+        if self.is_api_client_setup:
+            # this provider does not send NEW or RESERVED to datacite
+            if pid.is_registered():
+                delay = current_app.config[
+                    "RDM_RECORDS_DOI_DATACITE_UPDATE_DELAY"]
+                recid_value = record.pid.pid_value
+                doi_value = pid.pid_value  # pid (func param) is a DOI
+
+                doi_datacite_delete.apply_async(
+                    [recid_value, doi_value, self.client.name],
+                    countdown=delay
+                )
+        else:
+            logging.warning("DataCite client not configured. " +
```

Comment:
Minor: For error logging I don't remember what is the correct way, but in order to have errros properly aggregated in sentry, there's some special way of doing the formatting./

Reply:
In this case I wouldn't even add a logging. It DataCite is not setup, should we simply silently return?

Reply:
In any case it should `current_app.logger.warning(`.